### DISCUSSION TITLE
Add option for M2O fields to be displayed inline on parent form

### DISCUSSION THREAD

**Discussion by eliteproxy7** [directus/directus#5769] [Feature Requests]
It would be nice to be able to create a fieldset as templates, and apply them to collections, so if multiple collections will utilize the same fields, instead of having to re-create every single time we just associate that set of fields to the collection. 

either it creates the fields based on the template or creates the relation so i can add more to the field set template and now all my collections are have the set assigned have the new field also available

**Comment by benhaynes** [comment]
Well... That's exactly how many-to-one relationships work, which we already have! I think what you're looking for is a specific M2O interface that shows the fields inline within the parent form. That can be achieved within a custom extension, but I would also be down to consider adding that as a core interface.

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Comment by hanneskuettner** [comment]
You can use my interface extension for that purpose that should do exactly what you need. https://github.com/hanneskuettner/directus-extension-inline-form-interface
